### PR TITLE
Documentation for high DPI settings on Windows

### PR DESCRIPTION
**Pull request by AnHeuermann**
Adding some documentation on how to change DPI settings on tiny Windows notebooks with high-res screens.

### CODE REVIEW

File: doc/UsersGuide/source/omedit.rst [hunk 1]
```diff
@@ -1824,3 +1826,41 @@ the working directory from *Tools > Options > General* see section :ref:`omedit-
 
 For each simulation a new directory with the model name is created in the working directory and
 then all the simulation intermediate and results files are generated in it.
+
+
+High DPI Settings
+----------------------------------------------------
```
Comment (adeas31) [inline doc/UsersGuide/source/omedit.rst:1832]: Remove extra hyphens.

File: doc/UsersGuide/source/omedit.rst [hunk 2]
```diff
@@ -1824,3 +1826,41 @@ the working directory from *Tools > Options > General* see section :ref:`omedit-
...
+If you are using a high-resolution screen (1080p, 4k and more) and the app is blurry or
+the overall proportions of the different windows are off, it can help to change the DPI settings.
+
+On Windows it is possible to change the scaling factor to adjust the size of text, apps
+and other times, but the default setting might not be appropriate for OMEdit e.g., on
+compact notebooks with high resolution screens.
+
+You can either change the scaling factor for the whole Windows system or only change the
+scaling used for OMEdit. This is done by changing the `Compatibility` settings for
+`High DPI settings for OMEDit.exe` with the following steps:
```
Comment (adeas31) [inline doc/UsersGuide/source/omedit.rst:1846]: `OMEdit.exe` instead of `OMEDit.exe`

**Review by adeas31 — CHANGES_REQUESTED** [review]
LGTM. Just minor comments.

File: doc/UsersGuide/source/omedit.rst (hunk 1 above)
Comment (AnHeuermann) [reply doc/UsersGuide/source/omedit.rst]: Done

**Review by adeas31 — APPROVED** [review]

**Review by casella — APPROVED** [review]
LGTM.

In fact, I was not aware that you could set the dpi settings for a specific application. Good to know.  I guess you'll have to re-do this every time you install a new version of OMC, though.

**Pull request merged**